pkg/controller/nodeipam/ipam/cidrset/cidr_set.go

```diff
@@ -30,15 +30,24 @@ import (
 // be allocated from.
 type CidrSet struct {
 	sync.Mutex
+	// clusterCIDR is the CIDR assigned to the cluster
 	clusterCIDR     *net.IPNet
+	// clusterIP is the network IP of the cluster
 	clusterIP       net.IP
-	nodeMask        net.IPMask
+	// clusterMaskSize is the mask size, in bits, assigned to the cluster
 	clusterMaskSize int
+	// nodeMask is the network mask assigned to the nodes
+	nodeMask        net.IPMask
+	// subnetMaskSize is the mask size, in bits,assigned to the nodes
+	subNetMaskSize  int
```

Comment:
should this be called nodeMaskSize or nodeMaskBits or nodeMaskOnes?  We could remove it entirely and just let people reference it via `nodeMask.Size()` but that returns 2 values, so a bit more complicated.

Reply:
I was carrying over current names, and I was hesitating to check it because it was passed as parameter.
I prefer to use nodeMaskSize, it's more descriptive as you say.

Reply:
hmm calling Mask.Size() is expensive , 
```
howing nodes accounting for 8710ms, 56.74% of 15350ms total
Dropped 157 nodes (cum <= 76.75ms)
Showing top 10 nodes out of 105
      flat  flat%   sum%        cum   cum%
    1500ms  9.77%  9.77%     5080ms 33.09%  runtime.mallocgc
    1430ms  9.32% 19.09%     1430ms  9.32%  net.simpleMaskLength
    1080ms  7.04% 26.12%    11360ms 74.01%  k8s.io/kubernetes/pkg/controller/nodeipam/ipam/cidrset.(*CidrSet).AllocateNext
     840ms  5.47% 31.60%     1050ms  6.84%  runtime.heapBitsSetType
     810ms  5.28% 36.87%     7920ms 51.60%  k8s.io/kubernetes/pkg/controller/nodeipam/ipam/cidrset.(*CidrSet).indexToCIDRBlock
     730ms  4.76% 41.63%      820ms  5.34%  runtime.deferreturn
     700ms  4.56% 46.19%      700ms  4.56%  runtime.nextFreeFast
```

Reply:
If that's the case, I am fine with caching the size, just comment why, so I don't come along later and undo it :)